todo.js

```diff
@@ -5,6 +5,19 @@ const todoInput = document.querySelector('.todo-input')
 const todoItemsList = document.querySelector('.todo-items')
 
 
+const totalTasks = document.querySelector(".total-tasks span")
+const completedTasks = document.querySelector(".completed-tasks span")
+const remainingTasks = document.querySelector(".remaining-tasks span")
+ 
+function countTasks() {
+  console.log(todos.length,'TASK')
+  // totalTasks.textContent = todos.length
+  const completedTasksArray = todos.filter((item) => item.completed)
+  // console.log(completedTasksArray,'Show')
```

Comment:
remove commented/unwanted code
A comment should only be used to provide some info if pushed example
```//function depicting deletion of items```

Reply:
yes i will remove it